drivers/hd44780/include/hd44780_params.h

```diff
@@ -38,12 +40,31 @@ extern "C"
                HD44780_RW_OFF, HD44780_RW_OFF, HD44780_RW_OFF, HD44780_RW_OFF} \
 }
 
+#ifndef HD44780_PARAMS
+#define HD44780_PARAMS HD44780_PARAMS_ARDUINO
+#endif
+#endif
+
+#ifndef HD44780_PARAMS
+#define HD44780_PARAMS {            \
```

Comment:
Is this setup convenient for a particular board? If so, could you spare a comment mentioning it just for reference?

Reply:
No, it's just ascending numbers.
I wanted to make sure that overwriting the config with the board config works.